config/routes.rb

```diff
@@ -6,12 +6,16 @@
   resources :posts
   resources :comments, only: [:create]
```

Comment:
**SUGGESTION**

If you would like to delete comments by post or just have it easier to render them by post, you could nest the comments resource within posts;

```
resources :posts do
    resources :comments
end
```

To get paths such as:

`http://localhost:3000/posts/1/comments/1 `

and helped methods like:

`link_to "Delete comment", post_comment_path(post, comment), method: :delete`

More information [here](https://guides.rubyonrails.org/routing.html#nested-resources).